core/src/templates/components/card/card.twig

```diff
@@ -35,7 +47,7 @@
   {# Card Super Headline #}
   {%- block block_card_super_headline %}
     {% if card_super_headline is not empty %}
-      <h4>{{ card_super_headline }}</h4>
+      <span>{{ card_super_headline }}</span>
```

Comment:
Shouldn't this still be a header and an `h2` with the appropriate `type-x` style applied?

Reply:
I believe there was a discussion at some point, and the conclusion was that things that would appear in the "super headline" area probably shouldn't be treated as a HTML header element - usually things like category of the article. There's probably room for debate on this one though.

Reply:
Ok, I am fine with this, just wanted to check. Should we confirm with one other?

Reply:
Sure, perhaps @cjwest  or @buttonwillowsix?

Reply:
@cjwest , are you ok with me changing the "super headline" field from <h4> to a <span> so it doesn't cause problems with headline hierarchy (Level Access suggested that we shouldn't have a h4 above a h3)? I changed it to span since this is usually the place for metadata type info - category for example. We've also changed the headline to <h2>.

Reply:
Hi @yvonnetangsu,
Hmm.... I wonder if there is something we could use to be more semantic than <span>?

Reply:
> Hi @yvonnetangsu,
> Hmm.... I wonder if there is something we could use to be more semantic than ?

`span` is better than `div` at least 😄 , if you find anything better, let me know.

Reply:
Hi @yvonnetangsu,
I think you have it. I don't thing there's anything better than a `<span>`  or a `<p>`.
Would the `<p>`  make it easier to style?

Thanks for giving me a chance to look into it.

Reply:
Thanks, @cjwest ! Appreciate the 2nd pair of eyes. I was able to get the `span` to look ok no problem. Let's keep the `span` since it's short.